.github/workflows/publish-to-bcr.yaml

```diff
@@ -0,0 +1,40 @@
+# Publishes to the Bazel Central Registry.
+#
+# Based on .github/workflows/publish-to-bcr.yml from bazel-contrib/rules_scala v7.0.0.
+# See .bcr/README.md.
+name: Publish to the Bazel Central Registry
+
+on:
+  # Run from release.yml.
+  workflow_call:
+    inputs:
+      tag_name:
+        required: true
+        type: string
+    secrets:
+      bcr_publish_token:
```

Comment:
Please use `BCR_PUBLISH_TOKEN` for the secret name (all caps). All our secret names are named with all caps since they often get turned into environment variables, so it's nice to follow that convention even though that's not the case here.